frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -77,6 +103,17 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.VirtualMachinesDetailsPage),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: [`/k8s/ns/:ns/vmtemplates`, '/k8s/all-namespaces/vmtemplates'],
```

Comment:
Nit: no need for `` ` `` in static strings.

Reply:
:+1: